src/containers/select_guesthouse.jsx

```diff
@@ -3,20 +3,37 @@ import axios from 'axios';
 
 import Header from '../components/header.jsx';
 import TextBox from '../components/text_box.jsx';
-import SearchBar from '../components/search_bar.jsx';
+import SearchList from '../components/search_list.jsx';
 
 
 class SelectGuesthouse extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      db: {},
+      title: 'PAS',
+      tagLine: 'Park & Sleep',
```

Comment:
are title and tagLine ever change? if not, then its pointless to put them in a state.